src/App.css

```diff
@@ -790,6 +796,7 @@ body.dark {
   padding: 5px 10px;
   background-color: rgba(255, 255, 255, 0.8);
   display: inline-block;
+  text-transform: capitalize;
```

Comment:
This would modify the capitalization for all alerts, even ones that need to have uncapitalized words. I suggest capitalizing text in Javascript.

Reply:
Sure will do!